rfc000-rfc-process.md

```diff
@@ -124,6 +136,7 @@ RFC: unassigned
 Author: Alan Smithee <asmithee@example.com>
 Status: Draft
 Type: <Standards Track, Informational, Process>
+<Replaces: RFCxxx>
```

Comment:
The point is that the example template is supposed to be a copy-paste-go thing, not an exhaustive documentation. This will not be present on the vast majority of RFCs so it should not be in a the template.